### PR TITLE
challenge OG image (soon™️)

### PR DESCRIPTION
**Pull request by PickleNik**
needa fix fontz

### CODE REVIEW

**Comment by Hacksore** [comment]
@PickleNik you need me to pull up here?

**Comment by PickleNik** [comment]
> @PickleNik you need me to pull up here?

yes pls

**Comment by Hacksore** [comment]
image

**Comment by PickleNik** [comment]
image

**Comment by PickleNik** [comment]
image

**Comment by PickleNik** [comment]
Screenshot 2023-09-27 at 9 41 37 PM
deception

File: apps/og-image/public/amoguwuowoaahhh.png
Comment (Hacksore) [inline apps/og-image/public/amoguwuowoaahhh.png:1]: image

**Review by Hacksore — APPROVED** [review]

File: apps/og-image/public/amoguwuowoaahhh.png
Comment (PickleNik) [reply apps/og-image/public/amoguwuowoaahhh.png:1]: goodlookin

**Pull request merged**